app/grandchallenge/evaluation/templates/evaluation/submission_form.html

```diff
@@ -131,7 +131,7 @@ <h3>Create a new submission</h3>
                 {% endif %}
                 to this phase.
 
-                {% if phase.submission_kind == phase.SubmissionKindChoices.ALGORITHM %}
+                {% if phase.submission_kind == phase.SubmissionKindChoices.ALGORITHM and not phase.external_evaluation %}
```

Comment:
The participants need to know that their algorithm and model will be shared with the Challenge admins. They need to know that their algorithms and models will leave Grand Challenge, and we will have no control or insight into how often those containers are used, who by, and with what data. It needs explicit opt-in on the submission form with a checkbox.